visualize_translation_graph.py

```
@@ -0,0 +1,529 @@
+        mermaid = create_mermaid_diagram(graph)
+        mermaid_path = output_dir / "translation_graph.mmd"
+        mermaid_path.write_text(mermaid, encoding="utf-8")
+        print(f"✅ Saved Mermaid diagram: {mermaid_path}")
+        
+        # Also print to console
+        print("\n" + "=" * 60)
+        print("MERMAID DIAGRAM (copy to mermaid.live or GitHub)")
+        print("=" * 60)
+        print(mermaid)
+        print("=" * 60)
+    except Exception as e:
```

Ruff (0.14.0)

Do not catch blind exception: `Exception`

(BLE001)